index.bs

```diff
@@ -6008,7 +6009,16 @@ partial interface MLGraphBuilder {
     //   [[0, 0, 0],
     //    [9, 0, 0],
     //    [2, 6, 0]]
-    const lowerNegative = builder.triangular(input, { upper: false, diagonal: -1 });
+    const lowerNegative = builder.triangular(input, { upper: false, diagonal: -1 })
+
+    // lower triangular matrix with two batches:
+    //   [[[7, 0, 0],
+    //     [9, 4, 0],
+    //     [2, 6, 3]],
+    //    [[1, 0, 0],
+    //     [4, 5, 0],
+    //     [7, 8, 9]]]
+    const lower = builder.triangular(input, { upper: false });
```

Comment:
How about `lowerWithBatches` or `batchedLower`?

Reply:
`lowerWithBatches` SGTM - done!

Reply:
> "2-D" seems to be used more commonly than "2D" in the spec

That's amusingly quaint, because nobody really writes "2 dash D" anymore 😉.

"a rank-2 tensor does not usually describe a 2D space."
https://www.tensorflow.org/guide/tensor

"2D convolution layer (e.g. spatial convolution over images)."
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D

"where is the valid 2D operator"
https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html

"Real-World Examples of 0D, 1D, 2D, 3D, 4D and 5D Tensors"
https://medium.com/data-science-365/real-world-examples-of-0d-1d-2d-3d-4d-and-5d-tensors-100b0837ced4

"This 2D array or tensor ..."
https://wandb.ai/vincenttu/intro-to-tensors/reports/A-Gentle-Intro-To-Tensors-With-Examples--VmlldzozMTQ2MjE5

Reply:
Two dimensions? How quaint!

https://calormen.com/flatland/

Reply:
Spin up a PR to convert the spec and add to conventions?

Reply:
> Spin up a PR to convert the spec and add to conventions?

Sure, separately from this.